bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     certified: "false"
     createdAt: REPLACE_DATE
     description: An operator that enables Windows container workloads on OCP
-    olm.skipRange: '>=5.0.0 <6.0.0'
+    olm.skipRange: '>=6.0.0 <7.0.0'
```

Comment:
@mansikulkarni96 @saifshaikh48 I wonder if we can get `render-templates` to update all places where version is being used. Is that part of release improvement epic?